designs/UI/businesses.html

```diff
@@ -0,0 +1,106 @@
+        crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
+        crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+        crossorigin="anonymous"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl"
+        crossorigin="anonymous"></script>
+
+
+    <link href="css/main.css" rel="stylesheet">
+    <script src="js/main.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+
+</head>
+
+<body class="body">
+
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12 text-center">
+                <ul class="menu">
+                    <li>
+                        <a href="create-businesses.html">Create a Business</a>
+                    </li>
+                    <li>
+                        <a href="dashboard.html">My Profile</a>
+                    </li>
+                    <li class="active">
+                        <a href="index.html">Signout</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+
+<!--Start of business Listings -->
+<input type="text" id="myInput" onkeyup="myFunction()" placeholder="Search for names..">
+
+<table id="myTable">
+  <tr class="header">
+    <th style="width:25%;">Name</th>
+    <th style="width:25%;">Category</th>
+    <th style="width:25%;">Location</th>
+    <th style="width:25%;">Choose</th>
+  </tr>
+  <tr>
+    <td>Facebook</td>
+    <td>IT &amp; Networking</td>
+    <td>US</td>
+    <td><a href="f-dashboard.html" class="btn btn-primary icon  pull-right">Select</a></td>
+  </tr>
+  <tr>
+    <td>Toyota</td>
+    <td>Automotive</td>
+    <td>Kenya</td>
+    <td><a href="#" class="btn btn-primary icon  pull-right">Select</a></td>
+  </tr>
+  <tr>
+    <td>PAM Holdings</td>
+    <td>Real Estate</td>
+    <td>South Africa</td>
+    <td><a href="#" class="btn btn-primary icon  pull-right">Select</a></td>
+  </tr>
+  <tr>
+    <td>Jumia Travels</td>
+    <td>Hotels &amp;; Travel</td>
+    <td>Nigeria</td>
+    <td><a href="#" class="btn btn-primary icon  pull-right">Select</a></td>
+  </tr>
+</table>
+
+<!--End of Business listing -->
+
+    </div>
+    </div>
+    <div class="row" style="padding-top: 60px; clear: both;">
+        <div class="col-md-12 text-center">
+            <p>
+                <small>&copy; All Rights Reserved. Designed by
+                    <a href="https://victorkariuki.co.ke">Victor Kariuki</a>
+                </small>
+            </p>
+        </div>
+    </div>
+    </div>
+    
+    
+    
+    
+    </body>
+    
+    </html>
```

Comment:
It's highly advisable to a **one** extra line at the end of your file

Reply:
Understood